### ISSUE TITLE
Location based AR - Accuracy

### ISSUE THREAD

**Issue by robertocorrales** [Wikitude/wikitude-sdk-samples#28]
Hi, I would like to use your sdk to build an Geolocated AR experience for mobile. I have tow questions:
1.- Does the accuracy depends only on the mobile GPS?
2.- Does it have maps and 3D buildings to mask objects in real life with buildings?

Thanks!

**Comment by danielguttenberg** [comment]
1. You will also need a phone with a gyroscope. It will work on phones that don't have one by substituting with the accelerometer, but it won't be nearly as fluid and accurate an experience.

2. We do not provide any 3D models ourselves, you will have to acquire these elsewhere. Depending on what your exact use case is, GPS based AR might not be accurate enough to mask real life objects. You will have to try.

- Daniel

**Comment by robertocorrales** [comment]
OK, I understand that uses the gyroscope/accelerometer onces the object is positioned with latitude/longitude. And the engine uses gps/gyro/acce for everything. But is there a known issue regarding accuracy? Because GPS by itself, depending on the signal could have 2-5-10-20 meters of error from the point.
Is this the same? Or do you use other methods to place the objects in the exact lat/lon?
Thanks

Roberto

**Comment by danielguttenberg** [comment]
There is no additional tracking mechanism with Geo AR, you will be limited to a couple a meters depending on your GPS signal quality, as you correctly pointed out. 

We do offer visual tracking as well, but that is a topic entirely separate from Geo AR.

Daniel

**Comment by robertocorrales** [comment]
ok thanks!
Last question! Is it possible to combine GeoAr with visual tracking (buildings for example)?

**Comment by danielguttenberg** [comment]
Yes, you can combine them. You can have Geo AR in combination with any of our visual trackers. 

Daniel
